### PR TITLE
update pt-PT lang at servman

### PR DESCRIPTION
**Pull request by fgsoftware1**
< CAN YOU UPDATE THE DESCRIPTION INSTEAD OF KEEPING THE TEMPLATE TEXT? >

## Purpose

JIRA issue: [CORE-XXXX](https://jira.reactos.org/browse/CORE-XXXX)

## Proposed changes

_Describe what you propose to change/add/fix with this pull request._

- 
- 

## TODO

_Use a TODO when your pull request is Work in Progress._

- [ ] 
- [ ]

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,3 +12,5 @@ modules/[Aa][Hh][Kk]_[Tt]ests
 .settings
 .vscode
 sdk/tools/winesync/winesync.cfg
+/build
```
Comment (learn-more) [inline .gitignore:15]: Why?

File: base/applications/clipbrd/lang/pt-PT.rc [hunk 2]
```diff
@@ -0,0 +1,67 @@
+/* Translation and UTF-8 Conversion by Gabriel Aguiar(fgsoftware1) <fgygh5804@gmail.com,fgsoftwarestudio@gmail.com> (July, 2021) */
+
+LANGUAGE LANG_PORTUGUESE, SUBLANG_NEUTRAL
```
Comment (learn-more) [inline base/applications/clipbrd/lang/pt-PT.rc:3]: Did you test this?

File: .gitignore (hunk 1 above)
Comment (fgsoftware1) [reply .gitignore:15]: folder generated by cmake

File: base/applications/clipbrd/lang/pt-PT.rc (hunk 2 above)
Comment (fgsoftware1) [reply base/applications/clipbrd/lang/pt-PT.rc:3]: no

Comment (fgsoftware1) [reply base/applications/clipbrd/lang/pt-PT.rc:3]: i dont have reactos but im trying compile as i say above

Comment (learn-more) [reply base/applications/clipbrd/lang/pt-PT.rc:3]: > i dont have reactos but im trying compile as i say above

You did not mention that anywhere.

File: .gitignore (hunk 1 above)
Comment (learn-more) [reply .gitignore:15]: Not using our default build system.

Comment (fgsoftware1) [reply .gitignore:15]: i run configure script and build trought cmaketools on vscode

**Review by binarymaster — CHANGES_REQUESTED** [review]
This PR conflicts with your previous PR #3763, please finish it first.

**Comment by fgsoftware1** [comment]
no conflits

**Review by fgsoftware1 — COMMENTED** [review]
done

**Comment by binarymaster** [comment]
image

Repository is deleted, so this PR cannot be maintained.

**Pull request closed without merging**